modules/kargoAnalyticsAdapter.js

```diff
@@ -1,14 +1,71 @@
+import { _each, logError } from '../src/utils.js';
+import { ajax } from '../src/ajax.js';
 import adapter from '../src/AnalyticsAdapter.js';
 import adapterManager from '../src/adapterManager.js';
 
-var kargoAdapter = adapter({
-  analyticsType: 'endpoint',
-  url: 'https://krk.kargo.com/api/v1/event/prebid'
-});
+const EVENT_URL = 'https://krk.kargo.com/api/v1/event';
+const KARGO_BIDDER_CODE = 'kargo';
+const CONSTANTS = require('../src/constants.json');
+
+const analyticsType = 'endpoint';
+
+let _logData = {
+  auctionTimeout: 0,
+  bidId: '',
+};
+
+var kargoAnalyticsAdapter = Object.assign(
+  adapter({ analyticsType }), {
+    track({ eventType, args }) {
+      if (typeof args !== 'undefined') {
```

Comment:
Just stylistics:
```
var kargoAnalyticsAdapter = Object.assign(
  adapter({ analyticsType }),{
    track({ eventType, args }) {
      if (typeof args == 'undefined') {
        return;
      }

      switch (eventType) {
        case CONSTANTS.EVENTS.AUCTION_INIT: {
          _logData.auctionTimeout = args.timeout;
          break;
        }
        case CONSTANTS.EVENTS.BID_TIMEOUT: {
          handleTimeout(args);
          break;
        }
      }
    }
  }
);
```

Reply:
Yea good call - the undefined check was there with the original unfinished version of the adapter but it's not needed - just removed it.